openshift/launchpad-template.yaml

```diff
@@ -14,34 +14,136 @@ metadata:
 message: "The following service(s) have been created in your project: mission-control, launchpad-backend, launchpad-frontend.\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/openshiftio"
 labels:
   template: launchpad
+parameters:
+- name: LAUNCHPAD_MISSIONCONTROL_GITHUB_USERNAME
+  displayName: Your GitHub username
+  description: Your GitHub username (leave empty when using Keycloak)
+- name: LAUNCHPAD_MISSIONCONTROL_GITHUB_TOKEN
+  displayName: Your GitHub Mission Control access token
+  description: Your GitHub Mission Control access token (leave empty when using Keycloak)
+- name: LAUNCHPAD_MISSIONCONTROL_OPENSHIFT_API_URL
+  displayName: Target OpenShift API URL
+  description: The base URL of the OpenShift API where the launched boosters
+    should be created (ie. https://hostname[:port] or https://ipaddress[:port]).
+    Leave the default value when targetting the same OpenShift/Minishift you are
+    running this on
+  value: https://openshift.default.svc.cluster.local
+  required: true
+- name: LAUNCHPAD_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL
+  displayName: Target OpenShift Console URL
+  description: The base URL of the OpenShift Console where the launched boosters
+    should be created (ie. https://hostname[:port] or https://ipaddress[:port]).
+    When targetting the same OpenShift/Minishift instance you are running this on you
+    can most likely look at this page's address bar of your browser and copy the URL
+    Example value (https://$HOSTNAMEORIP:8443)
+  value:
+  required: true
+- name: LAUNCHPAD_MISSIONCONTROL_OPENSHIFT_USERNAME
+  displayName: OpenShift user name
+  description: OpenShift user name (leave empty when using Keycloak)
+  value: 
+  required: true
```

Comment:
The template will help Minishift users to get started quickly, so I guess it makes sense to have developer as the default value for the user and password